package.json

```diff
@@ -31,7 +31,8 @@
       "react"
     ],
     "plugins": [
-      "syntax-dynamic-import"
+      "syntax-dynamic-import",
+      "transform-object-rest-spread"
```

Comment:
This is to add support for the spread syntax ( `{... stuff}`) which is used by the map collar library.